src/control-plane/backend/config/dictionary.json

```diff
@@ -78,5 +78,237 @@
         "updateAt": 1667355960000
       }
     ]
+  },
+  {
+    "name": "MetadataBuiltInList",
+    "data": {
+      "PresetEvents": [
+        {
+          "name": "_first_open",
+          "description": "The first time a user opens an app"
```

Comment:
Should we support multiple languages for the description of preset events? @luorobin-a2z

Reply:
we can change the model as below,

```json
{
"description": {
  "zh-CN": "xxx",
  "en-US": "yyy"
}
}
```

Reply:
done